File: app/src/main/java/ru/divizdev/coinrate/Entities/v2/ApiData.java

package ru.divizdev.coinrate.Entities.v2;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ApiData {

    @SerializedName("data")
    @Expose
    private List<Datum> data = null;
    @SerializedName("status")
    @Expose
    private Status status;

    public List<Datum> getData() {
        return data;
    }

    public void setData(List<Datum> data) {
        this.data = data;
    }

    public Status getStatus() {
        return status;
    }

    public void setStatus(Status status) {
        this.status = status;
    }

}

File: app/src/main/java/ru/divizdev/coinrate/data/entities/ApiData.kt
package ru.divizdev.coinrate.data.entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ApiData(
    @SerializedName("data")
    @Expose
    var data: List<Data>? = null,

    @SerializedName("status")
    @Expose
    var status: Status? = null,
)